Source/Core/Core/FifoPlayer/FifoDataFile.cpp

```diff
@@ -166,6 +172,15 @@ std::unique_ptr<FifoDataFile> FifoDataFile::Load(const std::string& filename, bo
   file.Seek(header.xfRegsOffset, SEEK_SET);
   file.ReadArray(dataFile->m_XFRegs, size);
 
+  // Texture memory saving was added in version 4.
+  std::memset(dataFile->m_TexMem, 0, sizeof(m_TexMem));
```

Comment:
Why do you use 3 different sizes: sizeof(m_TexMem), TEX_MEM_SIZE and header.texMemSize ?